### ISSUE TITLE
faq.usps.com: breakage

### ISSUE THREAD

**Issue by Poke-The-Bear** [uBlockOrigin/uAssets#26146]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://faq.usps.com/s/article/USPS-Smart-Locker` 

### Category

breakage

### Description

contains a link to see a list of active locker locations, please visit the [Locker Location Page](https://www.uspssmartpackagelockers.com/locker-location). The link is blocked by:

Block Origin has prevented the following page from loading:

https://www.uspssmartpackagelockers.com/locker-location

Because of the following filter:

||uspssmartpackagelockers.com^$document,to=~usps.com
Found in:

    [uBlock filters – Badware risks](moz-extension://03d15ba8-0777-4101-8882-8b0f823ed132/asset-viewer.html?url=ublock-badware)

### Other extensions used

Google Analytics Blocker
NoScript
uBlock Origin

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>
usps page with link
uBlock page block

![image](https://github.com/user-attachments/assets/b5a75c27-e220-4c0f-bc5c-35b71a4e8f9b)
![image](https://github.com/user-attachments/assets/41753000-78c7-4aef-9e27-900b34fcd3ec)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.61.0
Firefox: 132
filterset (summary):
 network: 136916
 cosmetic: 49789
 scriptlet: 21396
 html: 2146
listset (total-discarded, last-updated):
 default:
  user-filters: 1-0, never
  ublock-filters: 40941-135, 20m Δ
  ublock-badware: 11982-1, 20m Δ
  ublock-privacy: 1510-22, 20m Δ
  ublock-unbreak: 2565-1, 20m Δ
  ublock-quick-fixes: 222-10, 20m Δ
  easylist: 79084-582, 20m Δ
  easyprivacy: 53273-682, 20m Δ
  urlhaus-1: 18715-0, 1h.20m
  plowe-0: 3530-0, now
filterset (user): [array of 1 redacted]
trustedset:
 added: [array of 3 redacted]
switchRuleset:
 added: [array of 2 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 840 ms (selfie)
 maxAssetCacheWait: 241 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 2
 no-cosmetic-filtering: true
 no-remote-fonts: true
 network:
  googletagmanager.com: 1
  uspssmartpackagelockers.com: 1
```
</details>

**Comment by stephenhawk8054** [comment]
Looks like false positive. @uBlock-user Sorry, can you remove it? I can't access my computer now.

**Comment by stephenhawk8054** [comment]
Yeah, remove `||uspssmartpackagelockers.com^$all,to=~usps.com` in "Badware list"

**Comment by stephenhawk8054** [comment]
Thanks

**Comment by Poke-The-Bear** [comment]
Thanks everyone!

**Comment by themaxgross** [comment]
I was going to open a new issue but found this. I think the false positive is back:

https://github.com/uBlockOrigin/uAssets/blob/ad5b677f4c6315aa1686a281325d2d47ecdafa21/filters/badware.txt#L4321
